pyproject.toml

```diff
@@ -135,6 +135,7 @@ test-requires = [
     "threadpoolctl",
     "pooch",
     "hypothesis",
+    "gmpy2",
```

Comment:
@andyfaff you wrote the comment above this list about missing `gmpy2` wheels. Was that referring to CPython 3.12? If so, is the addition here correct? If so, we should adjust the comment above here too.

Reply:
gentle ping @andyfaff